examples/flax/image-captioning/run_image_captioning_flax.py

```diff
@@ -1052,7 +1051,7 @@ def generate_step(params, batch):
     if not os.path.isdir(os.path.join(training_args.output_dir)):
         os.makedirs(os.path.join(training_args.output_dir), exist_ok=True)
 
-    def save_ckpt(ckpt_dir: str, commit_msg: str = ""):
+    def save_ckpt(ckpt_dir: str, commit_msg: str):
```

Comment:
We should keep the default here 

```suggestion
    def save_ckpt(ckpt_dir: str, commit_msg: str = ""):
```